charts/internal/calico/templates/kube-controller/deployment-calico-kube-controllers.yaml

```diff
@@ -27,6 +27,9 @@ spec:
       name: calico-kube-controllers
       namespace: kube-system
```

Comment:
@zanetworker  why are those there??

Reply:
I am not sure what you mean? if this is not related to this PR, can we discuss it in another thread, do you have any other comments on this PR?

Reply:
Okay, I see your point (why in the template you mean), probably because we copy the charts and customize the important values, it is like this [upstream](https://docs.projectcalico.org/manifests/calico.yaml). I am still not seeing how it is harmful though. In anycase, this is not related to this PR, so if you would like to change it, lets file another PR / issue for it.